src/main/asciidoc/java/http.adoc

```diff
@@ -1251,24 +1251,24 @@ client.websocket("/some-uri", websocket -> {
 
 ==== Writing messages to WebSockets
 
-If you wish to write a single binary WebSocket message containing a single WebSocket frame to the WebSocket (a
-common case) the simplest way to do this is to use `link:../../apidocs/io/vertx/core/http/WebSocket.html#writeMessage-io.vertx.core.buffer.Buffer-[writeMessage]`:
+If you wish to write a single binary WebSocket message to the WebSocket you can do this with
+`link:../../apidocs/io/vertx/core/http/WebSocket.html#writeBinaryMessage-io.vertx.core.buffer.Buffer-[writeBinaryMessage]`:
```

Comment:
I've found this a bit misleading in comparison to https://github.com/eclipse/vert.x/pull/1081/files#diff-1ee4c083f53d5cb366fbb5aa4adb042aR1296.

Maybe it needs some clarifications.